src/Microsoft.ML/Runtime/EntryPoints/ImportTextData.cs

```diff
@@ -27,13 +27,25 @@ public sealed class Input
             public string CustomSchema = null;
         }
 
+        [TlcModule.EntryPointKind(typeof(ILearningPipelineLoader))]
+        public sealed class LoaderInput
+        {
+            [Argument(ArgumentType.Required, ShortName = "data", HelpText = "Location of the input file", SortOrder = 1)]
+            public IFileHandle InputFile;
+
+            [Argument(ArgumentType.Required, ShortName = "args", HelpText = "Arguments", SortOrder = 1)]
+            public TextLoader.Arguments Arguments = new TextLoader.Arguments();
+        }
+
         public sealed class Output
         {
             [TlcModule.Output(Desc = "The resulting data view", SortOrder = 1)]
             public IDataView Data;
         }
 
-        [TlcModule.EntryPoint(Name = "Data.TextLoader", Desc = "Import a dataset from a text file")]
+#pragma warning disable 0618
+        [Obsolete("Use TextLoader instead.", false)]
```

Comment:
>, false [](start = 43, length = 7)

This is the default for the overload without it specified anyway, so it can be omitted.